interface/correctedLeptons.h

```diff
@@ -0,0 +1,37 @@
+#ifndef correctedLeptons_h
+#define correctedLeptons_h
+
+#include <vector>
+#include <utility>
+#include "bigTree.h"
+
+using vecf = std::vector<float>;
+
+class correctedLeptons { 
+public:
+  correctedLeptons(const unsigned, bigTree*);
+  std::pair<float, float> correctMET(float, float);
+  
+  float px(unsigned j) { return mPx[j]; };
+  float py(unsigned j) { return mPy[j]; };
+  float pz(unsigned j) { return mPz[j]; };
+  float en(unsigned j) { return mEn[j]; };
+
+  float metx() { return mMETx; };
+  float mety() { return mMETy; };
+  
+ private:
+  unsigned mSize = 0;
+  std::unique_ptr<bigTree> mBigTree;
```

Comment:
`bigTree* mBigTree;`

Reply:
Smart pointers are often incompatible with ROOT; I would have to dedicate a lot of time to understand why, and in this context it is simply not worth it.